1.0/docs/devguide/properties.md

```diff
@@ -150,7 +150,7 @@ names (for example, if a property is defined using `reflectToAttribute: true`.)
 
 **Compatibility note:** In 0.5, Polymer attempted to map attribute names to corresponding properties.
 For example, the attribute `foobar` would map to the property `fooBar` if it was
-defined on the element. This **does not happen in 0.8** &mdash; attribute to property
+defined on the element. This **does not happen in v1** &mdash; attribute to property
```

Comment:
Aren't we on 1.1 already? I guess 1.0 has slightly better connotations than v1, but it may make the docs feel a bit out of date too. WDYT?